cmd/config-versions.go

```diff
@@ -585,6 +586,8 @@ type serverConfigV22 struct {
 // IMPORTANT NOTE: When updating this struct make sure that
 // serverConfig.ConfigDiff() is updated as necessary.
 type serverConfigV23 struct {
+	quick.Config `json:"-"` // ignore interfaces
```

Comment:
this would move to `serverConfigV24` on rebase

Reply:
done